doop.c

```diff
@@ -635,6 +635,37 @@ Perl_do_trans(pTHX_ SV *sv)
     }
 }
 
+static int
+S_sv_has_magic_method(pTHX_ SV *sv, int method)
```

Comment:
This function should be added to embed.fnc, which will generate PERL_ARGS_SV_HAS_MAGIC macros, which should be the first line in the sub. Make sure you do a `make regen` aftewards.

Reply:
Also this should have some kind of pod docs for it.